incubator/clickhouse/values.yaml

```diff
@@ -0,0 +1,161 @@
+## Pod Annotations
+# podAnnotations: {}
+
+## Statefulset annotations
+# annotations: {}
+
+## Number of ClickHouse nodes
+replicas: 1
+
+## Image settings
+## ref: http://kubernetes.io/docs/user-guide/images/
+image: yandex/clickhouse-server
+imageTag: 18.12.17
+imagePullPolicy: IfNotPresent
```

Comment:
Can you please update the image format to follow our guidelines... https://github.com/helm/charts/blob/master/REVIEW_GUIDELINES.md#configuration